.github/workflows/build-and-test.yaml

```diff
@@ -32,5 +32,8 @@ jobs:
       - name: Test
         run: pnpm test
 
+      - name: Test types
+        run: pnpm --filter typed-openapi test:types
```

Comment:
You can also test agains [range of TypeScript versions](https://tstyche.org/guides/typescript-versions). To be sure older version work as expected (if that is important). For example:

```suggestion
        run: pnpm --filter typed-openapi test:types --target '>=5.2'
```